NEWS.txt

```diff
@@ -56,6 +56,7 @@ using the provided 'sstableupgrade' tool.
 
 New features
 ------------
+    - Tool to list data paths of existing tables
```

Comment:
I do not think this is necessary. It is not _tool_ as in a standalone tool, it is still the part of _nodetool_. Mentioning this in CHANGES.txt is enough.